scripts/deploy-backend.sh

```diff
@@ -160,7 +170,7 @@ else
 fi
 
 # Dynamically generate a new DJANGO_SECRET_KEY
-DJANGO_SECRET_KEY=$(python -c "from secrets import token_urlsafe; print(token_urlsafe(50))")
+DJANGO_SECRET_KEY=$(python3 -c "from secrets import token_urlsafe; print(token_urlsafe(50))")
```

Comment:
Was python3  only necessary locally? This seems to have worked in cloud gov so its probably fine?

Reply:
Twas used for local but yeah, looks like it was working via ci/cd.